torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -246,6 +246,41 @@ def from_float_static(
             dtype=input_float.dtype,
         )
 
+    @classmethod
+    def from_float_float(
```

Comment:
I think at this point `from_float`, `from_float_static`, `from_float_float` could use docblocks to explain how they differ from each other

Reply:
I'm still trying to decide on how do we call them? I don't want to add another call like https://github.com/pytorch/ao/blob/174e630af2be8cd18bc47c5e530765a82e97f45b/torchao/dtypes/affine_quantized_tensor.py#L921. I'm thinking of making a dict for calls based on target_dtype

Reply:
my 5c - writing docblocks for what is there now will help with deciding with naming, future refactors, etc.

Reply:
Ok, will do that

Reply:
also we may not need this if we can just reuse the `from_float` code path, this is a prototype to see how much overlap there is as well I think

Reply:
Yes, I don't want to update the from_float function before testing.